utils/src/deployer.rs

```diff
@@ -389,7 +410,7 @@ pub async fn deploy(
             .await?;
         let fee_contract = FeeContract::new(fee_contract_address, l1.clone());
         let data = fee_contract
-            .initialize(owner)
+            .initialize(deployer)
```

Comment:
the multisig can be set as the owner here since we do not want to call other functions on the fee contract with the deployer wallet, but lines 430 to 442 also works to accomplish that

Reply:
Yeah good point